app.js

```diff
@@ -35,14 +35,23 @@ import parseResults from './results.js';
 import {getStorage} from './storage.js';
 import {parseUA} from './ua-parser.js';
 import Tests from './tests.js';
+import {exec} from './scripts.js';
 
 const storage = getStorage();
 
 /* c8 ignore start */
-const appVersion =
-  process.env.NODE_ENV === 'production' ?
-    (await fs.readJson(new URL('./package.json', import.meta.url))).version :
-    'Dev';
+let appVersion;
+if (process.env.NODE_ENV === 'production') {
+  appVersion = (await fs.readJson(new URL('./package.json', import.meta.url)))
+    .version;
+} else {
+  try {
+    appVersion = 'Dev-' + String(exec('git describe --tags')).substr(1);
+  } catch (e) {
+    // If anything happens, i.e. git isn't installed, ignore
```

Comment:
I think let's instead fall back to package.json. So if `NODE_ENV !== 'production'` we try using `git describe --tags`, and if we don't get and return a version, we read it from package.json.